rules/ml/ml_rare_destination_country.toml

```diff
@@ -0,0 +1,38 @@
+[metadata]
+creation_date = "2021/04/05"
+maturity = "production"
+updated_date = "2021/04/05"
+
+[rule]
+anomaly_threshold = 75
+author = ["Elastic"]
+description = """
+A machine learning job detected an unusual destination country name in the network logs.
```

Comment:
Would "rare country" be a preferable term compared to "unusual"?

Reply:
Done!